xbmc/pvr/recordings/PVRRecording.h

```diff
@@ -82,8 +84,23 @@ namespace PVR
     std::string   m_strFanartPath;    /*!< fanart path */
     unsigned      m_iRecordingId;     /*!< id that won't change while xbmc is running */
 
+  private:
```

Comment:
Re friend class: ACK. I will come up with a solution without the friend class, most probably even without public ctor, because I'm still convinced that public ctor is not the right thing here and can be avoided.